dev_guide/deployments/deployment_strategies.adoc

```diff
@@ -52,28 +51,28 @@ downtime may be needed to complete the transition.
 - If you have a non-isolated test environment, you can break both new and old versions.
 
 
-Since the end user usually accesses the application through a route handled by a 
-router. The deployment strategy can focus on deployment configuration features or 
-routing features.
+Since the end user usually accesses the application through a route handled by a
+router. The deployment strategy can focus on deployment configuration features
+or routing features.
 
-Strategies that focus on the deployment configuration impact all routes that use the
-application. Strategies that use router features target individual routes.
+Strategies that focus on the deployment configuration impact all routes that use
+the application. Strategies that use router features target individual routes.
 
-Many deployment strategies are supported through the deployment configuration and
-some additional strategies are supported through router features. The deployment
-configuration based strategies are discussed in this section.
+Many deployment strategies are supported through the deployment configuration
+and some additional strategies are supported through router features. The
+deployment configuration based strategies are discussed in this section.
```

Comment:
How about "Strategies based on deployment configurations are discussed in this section."